aspnetcore/security/authentication/windowsauth.md

```diff
@@ -186,3 +186,7 @@ ASP.NET Core doesn't implement impersonation. Apps run with the app's identity f
 [!code-csharp[](windowsauth/sample_snapshot/Startup.cs?highlight=10-19)]
 
 `RunImpersonated` doesn't support asynchronous operations and shouldn't be used for complex scenarios. For example, wrapping entire requests or middleware chains isn't supported or recommended.
+
+### Claims transformations
+
+When hosting in-process, <xref:Microsoft.AspNetCore.Authentication.AuthenticationService.AuthenticateAsync*> isn't called internally to initialize a user. Therefore, an <xref:Microsoft.AspNetCore.Authentication.IClaimsTransformation> implementation used to transform claims after every authentication isn't activated by default. When transforming claims with an <xref:Microsoft.AspNetCore.Authentication.IClaimsTransformation> implementation, call <xref:Microsoft.Extensions.DependencyInjection.AuthenticationServiceCollectionExtensions.AddAuthentication*> to add authentication services. For a code example that activates claims transformations when hosting in-process, see <xref:host-and-deploy/aspnet-core-module#in-process-hosting-model>.
```

Comment:
"When hosting with IIS in-process mode,"

AddAuthentication doesn't tell you the required parameter to pass. This also needs to include UseAuthentication. 

How about simplifying this to only describe the problem and link to the other doc for the complete solution.

Reply:
Not a super UE given that it's the last point in a long list of in-process points of interest (and I dislike anchors :smile: ... inside joke with Rick ... he ❤️ them). Perhaps, we need a section for this content, then there will be something to link from here. I'll see how that might work.

Reply:
Nah ... let's just link. The list isn't that long. The reader will be able to find it. No anchor unless they force me at 🔪-point to do it.